pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, libusb, git }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchurl {
+    url = "https://github.com/medusalix/xow/archive/v${version}.tar.gz";
+    sha256 = "0a7621i4ig6wmd8gyq1lcfxgl4adbq8h2640yvj33cs858d6iz70";
+  };
+
+  makeFlags = "BUILD=RELEASE";
+  enableParallelBuilding = true;
+  buildInputs = [ libusb git ];
```

Comment:
`git` is used during the configure I assume? If so it should be in `nativeBuildInputs`. And you can probably also use `gitMinimal` instead.

Reply:
Works fine with `gitMinimal` inside `nativeBuildInputs`. Thanks!